### PR TITLE
RPR: Arcane Circle -> Plentiful Harvest & Enshroud -> Communio

### LINKED ISSUE #142: [suggestion] RPR Ability Combos
Further suggestions for combos to be implemented on RPR

Enshroud -> Communio
Communio can only be executed under the effect of Enshroud and ends the effect.

Arcane Circle -> Plentiful Harvest
Slightly more complicated. Arcane Circle allows for Plentiful Harvest 6s after arcane circle has been used, and can be executed as late as 30s after arcane circle. since Arcane Circle has a CD of 120s this means its impossible to be able to cast Plentiful Harvest while Arcane Circle isn't on cooldown.

### PR DESCRIPTION
**Pull request by Aelexe**
Replace Arcane Circle with Plentiful Harvest when Circle of Sacrifice or Immortal Sacrifice are active.
Replace Enshroud with Communio when Enshrouded is active.

Not able properly due to my build's buff checking still not functioning.

Resolves #142 and a portion of #113.

### CODE REVIEW

File: XIVComboPlugin/IconReplacer.cs [hunk 1]
```diff
@@ -1005,8 +1005,35 @@ private ulong GetIconDetour(byte self, uint actionID)
...
                     return RPR.SpinningScythe;
                 }
+            }
+
+            // Replace Arcane Circle with Plentiful Harvest when Circle of Sacrifice or Immortal Sacrifice are active.
+            if (Configuration.ComboPresets.HasFlag(CustomComboPreset.ReaperPlentifulHarvestFeature))
+            {
+                if (actionID == RPR.ArcaneCircle)
+                {
+                    if(level == RPR.Levels.PlentifulHarvest) {
```
Comment (kaedys) [inline XIVComboPlugin/IconReplacer.cs:1015]: Shouldn't this be `>=`?

File: XIVComboPlugin/IconReplacer.cs [hunk 2]
```diff
@@ -1005,8 +1005,35 @@ private ulong GetIconDetour(byte self, uint actionID)
...
+                }
+            }
 
-                return actionID;
+            // Replace Enshroud with Communio when Enshrouded is active.
+            if (Configuration.ComboPresets.HasFlag(CustomComboPreset.ReaperCommunioFeature))
+            {
+                if (actionID == RPR.Enshroud)
+                {
+                    if (level == RPR.Levels.Communio)
```
Comment (kaedys) [inline XIVComboPlugin/IconReplacer.cs:1029]: This should also probably be `>=`, simply for forwards compatibility.

**Comment by 7seven11even** [comment]
Does or will either of these work? If so where on the plug-in are they or how do I get them working?

**Comment by cozyGalvinism** [comment]
Why was this closed?

**Pull request closed without merging**